app/views/experience_stages/show.html.erb

```diff
@@ -91,15 +91,15 @@
               </div>
             </div>
 
-            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=show_all'>
+            <div ng-switch='theme_<%= theme.id %>_experiences' ng-init='theme_<%= theme.id %>_experiences=false'>
```

Comment:
Yeah - I think we've got a decent solution now. Basically, upon page load, all `theme_:id_experiences` are set to false, unless their `theme_:id` is in the url hash. Still hacky, but at least closer to the ideal UX. Let me know what you think: https://github.com/Sevenzo/challenge-platform/pull/11/files#diff-be8306af6458ecdbc31de0b171435d0dR5